packages/upset/src/components/Upset.tsx

```diff
@@ -59,24 +57,18 @@ export const Upset: FC<UpsetProps> = ({
     const defaultAttributes = (visualizeUpsetAttributes) ? DefaultConfig.visibleAttributes : [];
 
     /**
-     * visualizeAttributes can either be undefined or an array of strings
-     * if visualizeAttributes is undefined, load the first 3 attributes by default
+     * visualizeAttributes can either be undefined or an array of strings.
+     * if visualizeAttributes is undefined, load the first 3 attributes by default, else load none
```

Comment:
```suggestion
     * if visualizeAttributes is defined, load the attributes named within. Otherwise, load the first DEFAULT_NUM_ATTRIBUTES.
```

Reply:
Resolved in [bae2b9f](https://github.com/visdesignlab/upset2/pull/361/commits/bae2b9fcb606e3b81d781486a406bc9c2bb1c338)